tools/drdeploy.c

```diff
@@ -1404,7 +1417,15 @@ _tmain(int argc, TCHAR *targv[])
             /* support -dr_home alias used by script */
             strcmp(argv[i], "-dr_home") == 0) {
             dr_root = argv[++i];
-            read_tool_list(dr_root, dr_platform);
+            /* modify default toolconfig dir only.
```

Comment:
style nits to prefer capitalized complete sentences (old code doesn't always follow it but we'd like new code to do so):

s/modify/Modify/; s/the default/the default./